src/base/lineparser.cpp

```diff
@@ -948,3 +949,24 @@ bool LineParser::hasArg(int i) const
         return false;
     return true;
 }
+
+// Utility LineParser::ParseReturnValue converter to int
+int LineParser::getIntFromParseReturnValue(LineParser::ParseReturnValue value)
+{
+    int enumValue = value;
+    return enumValue;
+}
+// Utility LineParser::ParseReturnValue converter from int
+LineParser::ParseReturnValue LineParser::getParseReturnValueFromInt(int value)
+{
+    LineParser::ParseReturnValue valueEnum;
+
+    if (value == -1)
+        valueEnum = LineParser::ParseReturnValue::EndOfFile;
+    else if (value == 0)
+        valueEnum = LineParser::ParseReturnValue::Success;
+    else
+        valueEnum = LineParser::ParseReturnValue::Fail;
+
+    return valueEnum;
```

Comment:
Just condensing things a bit and removing the temporary:
```suggestion
    if (value == -1)
        return LineParser::ParseReturnValue::EndOfFile;
    else if (value == 0)
        return LineParser::ParseReturnValue::Success;

    return LineParser::ParseReturnValue::Fail;
```